Formula/sbt.rb

```diff
@@ -28,6 +28,16 @@ def install
       export JAVA_HOME="${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
       exec "#{libexec}/bin/sbt" "$@"
     EOS
+
+    (bin/"sbtn").write <<~EOS
+      #!/bin/sh
+      if [ -f "$HOME/.sbtconfig" ]; then
+        echo "Use of ~/.sbtconfig is deprecated, please migrate global settings to #{etc}/sbtopts" >&2
+        . "$HOME/.sbtconfig"
```

Comment:
This should be implemented upstream

Reply:
It's here because it's already in the other generated executable. Do you want me to remove it from both?

Reply:
Yeah, that seems better.

Reply:
I don't feel familiar enough with SBT internals to be confident about removing this. Would love some feedback from somebody more knowledgeable about SBT, e.g. from @eed3si9n who added these lines in the first place, about the original purpose of this check and how upstream deals with this situation.

Reply:
I don't think I'm the original author on the `Use of ~/.sbtconfig is deprecated`, or at least it was already there in 2016 - https://github.com/Homebrew/homebrew-core/pull/3064/commits/282d3fe0b59d874450105774f95d7b6b97d11b9e. Regardless, I think it's fine to remove the warning. The more Homebrew can be closer to the normal installation the better.

Reply:
hmm should have looked harder. here's the original original culprit: 008f55af1a92. Glad to have your blessing, I'll remove it. thanks!